tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
-  "files": [],
-  "references": [
-    { "path": "./tsconfig.app.json" },
-    { "path": "./tsconfig.node.json" }
-  ]
+  "compilerOptions": {
+    "lib": ["es2016", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["src/*"],
```

Comment:
Please check if its working , if yes then use at above

Reply:
Yes, it is working. Changed the path